src/ng/directive/input.js

```diff
@@ -1647,7 +1647,8 @@ function rangeInputType(scope, element, attr, ctrl, $sniffer, $browser) {
       // IE11 doesn't set the el val correctly if the maxVal is greater than the element value
       if (maxVal < elVal) {
         element.val(maxVal);
-        elVal = minVal;
+        // IE11 and Chrome don't set the value to the minVal when max < min
+        elVal = maxVal < minVal ? minVal : maxVal;
```

Comment:
Shouldn't this be reflected to `element.val()` then? I would expect something like:

```js
if (maxVal < elVal) {
  elVal = Math.max(minVal, maxVal);
  element.val(elVal);
}
```